rest/routing.go

```diff
@@ -293,7 +295,18 @@ func CreateAdminRouter(sc *ServerContext) *mux.Router {
 	// The routes below are part of the CouchDB REST API but should only be available to admins,
 	// so the handlers are moved to the admin port.
 	r.Handle("/{newdb:"+dbRegex+"}/",
-		makeHandler(sc, adminPrivs, []Permission{PermCreateDb}, nil, (*handler).handleCreateDB)).Methods("PUT")
+		makeHandlerSpecificAuthScope(sc, adminPrivs, []Permission{PermCreateDb}, nil, (*handler).handleCreateDB, func(bodyJSON []byte) (string, error) {
+			var body db.Body
```

Comment:
Any reason to use db.Body here, as opposed to a new struct that just has "bucket" defined?

Reply:
Was just quick and dirty. Was going to change that on Monday after approval of basic idea.